runner/custom_template_factory.py

```diff
@@ -289,8 +310,15 @@ def get_scene_file(parameters_file: str) -> str:
 
     return scene_file
 
-def get_dedicated_vm_count(parameters_file: str) -> str:
 
+def get_dedicated_vm_count(parameters_file: str) -> str:
+    """[summary]
+    
+    :param parameters_file: Path of the parameters file
```

Comment:
I like seeing this consistent type of style

Reply:
[summary].. oops. fixed